actioncable/lib/action_cable/engine.rb

```diff
@@ -24,11 +24,12 @@ class Railtie < Rails::Engine # :nodoc:
       options = app.config.action_cable
       options.allowed_request_origins ||= "http://localhost:3000" if ::Rails.env.development?
 
-      app.paths.add "config/redis/cable", with: "config/redis/cable.yml"
+      app.paths.add "config/cable", with: "config/cable.yml"
 
       ActiveSupport.on_load(:action_cable) do
-        if (redis_cable_path = Pathname.new(app.config.paths["config/redis/cable"].first)).exist?
-          self.redis = Rails.application.config_for(redis_cable_path).with_indifferent_access
+        if (config_path = Pathname.new(app.config.paths["config/cable"].first)).exist?
+          # TODO: remove this comment -- anyway, self is an instance of ActionCable::Server::Configuration
+          self.config_opts = Rails.application.config_for(config_path).with_indifferent_access
```

Comment:
`config_opts` and `@server.config.config_opts` reads off. Can `@server.config.cable` work?